CompuCell3D/core/CompuCell3D/Potts3D/Potts3D.cpp

```diff
@@ -1381,12 +1381,14 @@ void Potts3D::update(CC3DXMLElement *_xmlData, bool _fullInitFlag) {
 
             //we do not allow steering for motility specified by type
             sim->ppdCC3DPtr->temperature = _xmlData->getFirstElement("FluctuationAmplitude")->getDouble();
+            temperature = _xmlData->getFirstElement("FluctuationAmplitude")->getDouble();
             fluctAmplGlobalReadFlag = true;
         }
     }
 
     if (!fluctAmplGlobalReadFlag && _xmlData->getFirstElement("Temperature")) {
         sim->ppdCC3DPtr->temperature = _xmlData->getFirstElement("Temperature")->getDouble();
+        temperature = _xmlData->getFirstElement("Temperature")->getDouble();

```

Comment:
Why do we need two variables for temperature? We could use a comment for this.

Reply:
Probably one for @maciekswat to address. This one pre-dates me.